README.org

```diff
@@ -18,3 +18,6 @@ Here's a table describing the bindings
 | =ediff-suspend=             | =z=              | =C-z=          |
 | scroll left                 | =>=              | =zh=           |
 | scroll right                | =<=              | =zl=           |
+| copy A region to B's region | a                | l              |
+| copy B region to A's region | b                | h              |
```

Comment:
We can add l and h as additional options when it's just two files. That's ok with me. I see you want `l` to mean move from left to right I guess?